assets/prefabs/cultures/default.prefab

```diff
@@ -1,8 +1,8 @@
 {
     "Culture" : {
         "name" : "develop clan",
-        "buildingNeedPerZone" : {"UTILITY" : 2, "GOVERNMENTAL" : 1, "CLERICAL" : 1, "RESIDENTIAL" : 12, "COMMERCIAL" : 6},
-        "growthRate" : 500,
+        "buildingNeedPerZone" : {"UTILITY" : 0.08, "GOVERNMENTAL" : 0.5, "CLERICAL" : 0.5, "RESIDENTIAL" : 1.1, "COMMERCIAL" : 0.7},
```

Comment:
These are changes from your previous PR. An extra commit snuck into this one.

Reply:
Whoops! I gotta pay more attention to my commits next time. Hopefully this commit didn't slip into any other PRs as well 😄